service/access.go

```diff
@@ -347,6 +364,30 @@ func (a *accessService) updateAccessToken(ctx context.Context, domain, role, pro
 	return at.(*AccessTokenResponse), err
 }
 
+func (a *accessService) storeTokenCache(key string, acd *accessCacheData, expTimeDelta time.Time, expTime *jwt.NumericDate) {
+	oldTokenCacheData, _ := a.tokenCache.Get(key)
+	a.tokenCache.SetWithExpire(key, acd, expTime.Sub(expTimeDelta))
+	if oldTokenCacheData != nil {
+		if oldTokenCache, ok := oldTokenCacheData.(*accessCacheData); ok {
+			oldTokenCacheSize := accessCacheMemoryUsage(oldTokenCache)
+			a.memoryUsage += accessCacheMemoryUsage(acd) - oldTokenCacheSize
+			return
+		}
+		a.memoryUsage += accessCacheMemoryUsage(acd)
+		a.memoryUsage += int64(len(key))
+		return
```

Comment:
As old object exists, we do not need the key's size.
```suggestion
		a.memoryUsage += accessCacheMemoryUsage(acd)
		return
```

Reply:
Can you add a unit test for this?

1. cache.Set(some string)
2. cache.Set(some accessCacheDate)

Reply:
@WindzCUHK Added test case in https://github.com/AthenZ/athenz-client-sidecar/pull/18/commits/2e365199c60a3ad2e4974b7b60e2c466aaac35dd